src/compiler/Parser/ParsedHierarchy/SymbolType.ts

```diff
@@ -1,9 +1,9 @@
 export enum SymbolType {
```

Comment:
All of this was wrong also. The order of all these is only used in the CollisionChecking code, which meant that a lot of collision were not actually tested. This now behind us.